### PR TITLE
Add Captain's Maneuver

### PR DESCRIPTION
**Pull request by sprangg**
This commit adds the card Captain's Maneuver to Apocalypse and Planechase. Used [Carom](https://github.com/magefree/mage/blob/master/Mage.Sets/src/mage/cards/c/Carom.java) as the main reference on how to implement this. Tested with all permutations redirecting to/from a creature, a planeswalker, and a player.

### CODE REVIEW

**Comment by sprangg** [comment]
Fixes half of https://github.com/magefree/mage/issues/8788

**Review by awjackson — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/c/CaptainsManeuver.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    protected MageObjectReference redirectToObject;
+
+    public CaptainsManeuverEffect() {
+        super(Duration.EndOfTurn, Integer.MAX_VALUE, UsageType.ONE_USAGE_ABSOLUTE);
+        staticText = "The next X damage that would be dealt to target creature, planeswalker, or player this turn is dealt to another target creature, planeswalker, or player instead.";
+    }
+
+    public CaptainsManeuverEffect(final CaptainsManeuverEffect effect) {
+        super(effect);
```
[the remark this answers is not in the document]
Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/c/CaptainsManeuver.java:64]: Also, the copy constructor should be private (for consistence reasons).

[the remark this answers is not in the document]
Comment (sprangg) [reply Mage.Sets/src/mage/cards/c/CaptainsManeuver.java:64]: Many thanks for the reviews guys! These fixes are now added.

File: Mage.Sets/src/mage/cards/c/CaptainsManeuver.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+ */
+public final class CaptainsManeuver extends CardImpl {
+
+    public CaptainsManeuver(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{X}{R}{W}");
+
+        //The next X damage that would be dealt to target creature, planeswalker, or player this turn is dealt to another target creature, planeswalker, or player instead.
+        this.getSpellAbility().addEffect(new CaptainsManeuverEffect());
+
+        FilterCreaturePlayerOrPlaneswalker filter = new FilterCreaturePlayerOrPlaneswalker("creature, planeswalker or player (damage is redirected from)");
```
[the remark this answers is not in the document]
Comment (sprangg) [reply Mage.Sets/src/mage/cards/c/CaptainsManeuver.java:31]: I'll keep that in mind for the next time, thanks

**Review by awjackson — APPROVED** [review]

**Pull request merged**